docs/_snippets/storybook-addon-a11y-component-config.md

```diff
@@ -24,6 +24,10 @@ const meta: Meta<MyComponent> = {
         ],
       },
       options: {},
+    },
+  },
+  globals: {
```

Comment:
@valentinpalkovic — Story globals are only available behind a feature flag for Viewport and Backgrounds addons. Do you foresee any issues that this addon is using that API without a feature flag?

Reply:
Viewport and backgrounds had some more complex logic implemented. It is not a problem at all for a11y.